src/mjolnir/util.cc

```diff
@@ -768,6 +843,8 @@ bool build_tile_set(const boost::property_tree::ptree& original_config,
   // Validate the graph and add information that cannot be added until full graph is formed.
   if (start_stage <= BuildStage::kValidate && BuildStage::kValidate <= end_stage) {
     GraphValidator::Validate(config);
+    // set the checksum of the PBF files for all tiles
+    set_checksums(input_files, tile_dir);
```

Comment:
i feel like there is a big catch here. because we allow resuming tile builds and some people will even fan out different stages of the tile build on different sets of machines to speed them up, we cant rely on the input files actually still being around by the valid stage of the build. what i think we might have to do is compute the checksum for input as soon as we are dealing with it and carry it along somehow. i didnt think too hard about it so i could be totally wrong!

Reply:
could just write it in the temp files like we do with the sequences and then graph builder phase can set it?

Reply:
Oops yeah, good point😅